src/ParameterDistributions.jl

```diff
@@ -154,11 +163,16 @@ function bounded(lower_bound::FT, upper_bound::FT) where {FT <: Real}
             c_to_u = (x -> log((x - lower_bound) / (upper_bound - x)))
             jacobian = (x -> 1.0 / (upper_bound - x) + 1.0 / (x - lower_bound))
             u_to_c = (x -> upper_bound - (upper_bound - lower_bound) / (exp(x) + 1))
-            return Constraint(c_to_u, jacobian, u_to_c)
+            parameters = Dict("lower_bound" => lower_bound, "upper_bound" => upper_bound)
+            return Constraint{Bounded}(c_to_u, jacobian, u_to_c, parameters)
         end
     end
 end
 
+get_parameters(c::CType) where {CType <: ConstraintType} = c.parameters
```

Comment:
similar here, maybe rename to `get_bounds`.

Reply:
done